dns/py/data.py

```diff
@@ -20,6 +20,7 @@
 import sqlite3
 
 from params import PARAMETERS
+from functools import reduce
```

Comment:
where is reduce used?

Reply:
It's being used here: https://github.com/kubernetes/perf-tests/blob/master/dns/py/data.py#L103-L104

But, the import was present within the context of that function. I used the 2to3 tool for some of the conversions so this might have come through from there. Have removed it.